source/dagger/core/graphics/sprite.h

```diff
@@ -21,8 +37,16 @@ namespace dagger
 		ViewPtr<Shader> shader{ Shader::s_FirstLoadedShader };
 	};
 
-	void AssignSpriteTexture(Sprite& spriteTarget_, String textureName_);
-	void AssignSpriteTexture(Sprite& spriteTarget_, ViewPtr<Texture> texture_);
+	struct Spritesheet
```

Comment:
I don't have a proper name for this, but seems like Spritesheet is a bit misleading

Reply:
Agreed! Let's think about this collectively :D

Reply:
SpriteFrame!